include/openrave/planner.h

```diff
@@ -491,12 +491,14 @@ class OPENRAVE_API PlannerStatus
     PlannerStatus(const std::string& description, const int statusCode, CollisionReportPtr report=CollisionReportPtr());
     PlannerStatus(const std::string& description, const int statusCode, const IkParameterization& ikparam, CollisionReportPtr report=CollisionReportPtr());
     PlannerStatus(const std::string& description, const int statusCode, const std::vector<dReal>& jointValues, CollisionReportPtr report=CollisionReportPtr());
+    
     virtual ~PlannerStatus();
 
     PlannerStatus& SetErrorOrigin(const std::string& errorOrigin);
     PlannerStatus& SetPlannerParameters(PlannerParametersConstPtr parameters);
 
     void SaveToJson(rapidjson::Value& rPlannerStatus, rapidjson::Document::AllocatorType& alloc) const;
+    void SaveToJson2(rapidjson::Value& rPlannerStatus, rapidjson::Document::AllocatorType& alloc) const;
```

Comment:
What's this `SaveToJson2`? Please merge your additions into existing `SaveToJson`

Reply:
Okay, will do